edge/pkg/devicetwin/dtmanager/dmiworker.go

```diff
@@ -20,11 +20,10 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"k8s.io/klog/v2"
```

Comment:
Please change the order of import, just leave it as it is

Reply:
ok，I have made it.